forum/qa-plugin/user-activity-log/events-window/events-info-scripts.js

```diff
@@ -1,15 +1,22 @@
 const openWindow = document.querySelector('.open-modal');
 if(openWindow) {
-    const modal = document.querySelector('.modal-content');
+    const modal = document.querySelector('.modal-content').classList;
```

Comment:
Skoro zmienna `modal` teraz przechowuje listę klas elementu, a nie sam element, to powinna mieć odpowiednio dopasowaną nazwę, np. `modalClassList`. Wtedy patrząc na użyte później metody `remove` i `add` będzie bardziej zrozumiałe, że chodzi o manipulację klasami, a nie usunięcie czy dodanie elementu.